lib/src/utils/utils.dart

```diff
@@ -27,3 +28,14 @@ Size getDefaultSize(BuildContext context) {
 double translateRotatedPosition(double size, double degree) {
   return (size / 4) * math.sin(radians(degree.abs()));
 }
+
+/// takes in a path and returns a dathed path
+/// data.dashArray is provided
+/// dynamic because dart has no union types
+Path pathDasher(Path path, Paint painter, dynamic data) {
```

Comment:
Why here we take dynamic?
Why we don't use the strong typed `CircularArray`?

Reply:
I'll pass in the array next

Reply:
done